### DISCUSSION TITLE
S2 feature parity with S1 planned?

### DISCUSSION THREAD

**Discussion by cheema-corellian** [adobe/react-spectrum#7433] [General]
I occasionally peek at the S2 preview storybook (https://react-spectrum.adobe.com/s2/) to see how things are coming along on this upcoming release.  And it is looking good.  Some critical components from S1 are still missing however.  Like Date/Time related components.

Is it correct to assume that the initial public release of S2 will not have feature parity with S1?  And that the missing components will get added at a later time?

**Comment by dannify** [comment]
We don't have a date for public release of S2 but I can tell you that the date/time components will be included. We are working with our design team on them currently.

**Reply by cheema-corellian** [reply]
Is feature parity with S1 a goal for initial public release of S2?

Also, S2 public release was initially planned for 2024.  Is it correct to assume that 2025 is now more likely?

**Reply by devongovett** [reply]
It pretty much is public already - you're looking at it and it is on npm for anyone to use. If you're asking about a 1.0, yes that will be in 2025.

Feature parity is hard to define because they are different designs with different features but the goal is to support most of the existing S1 components at least (minus some that may be deprecated).
